docs/sources/configure-client/grafana-agent/ebpf/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Profiling with eBPF with Grafana Agent"
+menuTitle: "Profiling with eBPF"
+description: "Learn about using eBPF for continuous profiling for performance optimization."
+weight: 20
+aliases:
+  - /docs/phlare/latest/configure-client/language-sdks/ebpf/
+  - /docs/pyroscope/next/configure-client/language-sdks/ebpf
+---
+
+# Profiling with eBPF with Grafana Agent
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b0/EBPF_logo.png/240px-EBPF_logo.png" width="100px;" alt="eBPF"/>
```

Comment:
I'm not sure about using the logo here. It's probably okay to use the image, but I'm not sure if we should upload it to the Grafana servers or call it from the wikimedia site directly. I'll find out and let you know.

Reply:
Okay, looks like we can use the logo, we just have to upload it to our server. I'll do that and update the URL.

Reply:
```suggestion
<img src="/media/docs/pyroscope/ebpf_logo_color_on_white.png" width="100px;" alt="eBPF"/>
```